modules/accredited_representative_portal/db/migrate/20250128162530_create_ar_power_of_attorney_form_submissions.rb

```diff
@@ -0,0 +1,14 @@
+class CreateArPowerOfAttorneyFormSubmissions < ActiveRecord::Migration[7.2]
+  def change
+    create_table :ar_power_of_attorney_form_submissions do |t|
+      t.uuid :power_of_attorney_request_id, null: false
+      t.string :service_id
+      t.jsonb :service_response_ciphertext
+      t.string :status, default: 'pending'
+      t.datetime :status_updated_at
+      t.text :error_message_ciphertext
```

Comment:
Are we settled on this single error message or do we want a `jsonb` complying with JSON:API spec's error object? "Errors" section of [ticket](https://github.com/department-of-veterans-affairs/va.gov-team/issues/101919) talks about what would populate this data.

Reply:
If all that is true maybe `service_response_ciphertext` would be `text` also.

Reply:
And for your broader point that going with `text` wouldn't stop us from saving json, I agree. The only remaining contention then would be the name `error_message`, but again, I think we don't need to convince ourselves that the more complex thing needs to be set in stone right now.